content/en/security/logs.md

```diff
@@ -62,6 +44,17 @@ Additionally, certain features are not available at the moment to customers who
 
 If you have any questions about how the Log Management Service satisfies the applicable requirements under HIPAA, please contact your account manager.
 
+**Note:**
+
+HIPAA-enabled custommer used to require the use of specific endpoints to submit logs in order to enforce some specific encryptions. This is not needed anymore as that those encryptions are now enabled on all log submission endpoints.
+
+Here are the historical endpoints which are now deprecated : 
```

Comment:
Deprecated is not the right word here as we are still maintaining them. Maybe `legacy` would be better? @atspangler what do you think?